src/Nethermind.Int256/UInt256.cs

```diff
@@ -25,7 +26,11 @@ namespace Nethermind.Int256;
     // Ensure that hashes are different for every run of the node and every node, so if are any hash collisions on
     // one node they will not be the same on another node or across a restart so hash collision cannot be used to degrade
     // the performance of the network as a whole.
-    private static readonly uint s_instanceRandom = (uint)System.Security.Cryptography.RandomNumberGenerator.GetInt32(int.MinValue, int.MaxValue);
+    // Constant by default for zkVM-compatibility.
+    private static readonly uint _hashSeed =
+        AppContext.TryGetSwitch("Nethermind.Int256.UseRandomSeed", out var useRandomSeed) && useRandomSeed
```

Comment:
Shouldn't this be inverted so is random by default and can be switched off; rather than off by default?